mllib-dal/src/main/native/CorrelationImpl.cpp

```diff
@@ -149,13 +149,16 @@ static void doCorrelationDaalCompute(JNIEnv *env, jobject obj, size_t rankId,
 
 #ifdef CPU_GPU_PROFILE
 static void doCorrelationOneAPICompute(
-    JNIEnv *env, jlong pNumTabData,
+    JNIEnv *env, jlong pNumTabData, jlong numRows, jlong numClos,
     preview::spmd::communicator<preview::spmd::device_memory_access::usm> comm,
-    jobject resultObj) {
+    jobject resultObj, sycl::queue &queue) {
```

Comment:
could you make queue a private member of comm, so that can be got from comm without change the interface?

Reply:
the interfaces is still changed, could you remove queue in the interface?